.github/workflows/onPushToMain.yml

```diff
@@ -9,36 +9,32 @@ jobs:
     uses: salesforcecli/github-workflows/.github/workflows/unitTest.yml@main
   release:
     runs-on: ubuntu-latest
-    container:
-      image: node:lts
```

Comment:
Based on some light [research](https://github.com/actions/checkout/issues/363#issuecomment-1073731046), I think this was our likely culprit.

Reply:
good hunting